src/cinder/gl/Fbo.cpp

```diff
@@ -305,7 +305,7 @@ void Fbo::initMultisamplingSettings( bool *useMsaa, bool *useCsaa, Format *forma
 }
 
 // Iterate the Format's requested attachments and create any we don't already have attachments for
-void Fbo::prepareAttachments( const Fbo::Format &format, bool multisampling )
+void Fbo::prepareAttachments( const Fbo::Format &format, bool /*multisampling*/ )
```

Comment:
So it seems like there is some future work in progress to better support attachments with multisampling and some other configurations. In light of that, I'm going to accept this and just write a TODO there, which will likely get blown away by a work in progress overhaul.